sds_data_manager/lambda_code/SDSCode/instruments/l1b_Codice.py

```diff
@@ -0,0 +1,37 @@
+"""Lambda runtime code that triggers off of arrival of data into S3 bucket.
+"""
+import os
+from datetime import datetime
+
+import boto3
+
+#TODO: ability to access database, EFS, calibration data, etc.
+
+def lambda_handler(event: dict, context):
+    """Handler function"""
+    print(event)
+    print(context)
+
+    now = datetime.now()
+    print("Now time is")
+    print(now)
+
+    # Get the environment variables
+    bucket = os.environ['PROCESSING_PATH']
```

Comment:
I didn't know `PROCESSING_PATH` was bucket name. can we name it to `S3_BUCKET` or something?